functions/packages.bash

```diff
@@ -297,7 +301,7 @@ mqtt_setup() {
       echo -e "\\npassword_file ${mosquittoPasswd}\\nallow_anonymous false\\n" >> ${mosquittoConf}
     fi
     touch ${mosquittoPasswd}
-    chown "mosquitto:${username:-openhabian}" ${mosquittoPasswd}
+    chown "mosquitto:${groupname:-openhabian}" ${mosquittoPasswd}
```

Comment:
no such thing as groupname

Reply:
ups. Don't know what I was doing.
Do you want this to be replaced by "mosquitto:openhab"? 
With the expansion it will become group openhabian.